README.rst

```diff
@@ -24,6 +24,18 @@
   :target: https://doi.org/10.5281/zenodo.1012598
   :alt: Zenodo (latest)
 
+.. image :: https://repology.org/badge/version-for-repo/debian_unstable/heudiconv.svg?header=Debian%20Unstable
```

Comment:
```suggestion
.. image:: https://repology.org/badge/version-for-repo/debian_unstable/heudiconv.svg?header=Debian%20Unstable
```

That's where I "didn't see it" since it was greyed out as comment

Reply:
fixed in [3d61b9b](https://github.com/nipy/heudiconv/pull/632/commits/3d61b9be39e5c1c26ab73ffe6bbf28746220b291)